metpetdb_api/install_metpetdb.sh

```diff
@@ -0,0 +1,323 @@
+#!/bin/bash
```

Comment:
This should be in separate PR. Also, I'd like to receive credit for writing this.